### PR TITLE
small dockerFile cleanup.

### PR DESCRIPTION
**Pull request by rubemlrm**
Joined apt-get update with rest of apt-get stuff to free a layer and reduce docker size
Added cleanup command so after container setup we remove package archive and save some container space

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,13 +7,14 @@
...
 RUN echo "mysql-server mysql-server/root_password password \"''\"" | debconf-set-selections && \
     echo "mysql-server mysql-server/root_password_again password \"''\"" | debconf-set-selections
-RUN apt-get install --yes build-essential mysql-server postgresql libgd-dev libxml2 libxslt1-dev \
+RUN apt-get update && apt-get update --fix-missing && \
+    apt-get install --yes build-essential mysql-server postgresql libgd-dev libxml2 libxslt1-dev \
     libtidy-dev libreadline6 gettext libfreetype6 git autoconf bison re2c openssl pkg-config libssl-dev \
     libbz2-dev libcurl4-openssl-dev libenchant-dev libgmp-dev libicu-dev libmcrypt-dev \
-    postgresql-server-dev-all libpspell-dev libreadline-dev gnupg wget
+    postgresql-server-dev-all libpspell-dev libreadline-dev gnupg wget \
+    && rm -rf /var/lib/apt/lists/*
```
Comment (dshafik) [inline Dockerfile:17]: What is the purpose of the `rm` here?

Comment (dshafik) [reply Dockerfile:17]: Nevermind, I see the rest of the PR title now!

**Comment by dshafik** [comment]
Thanks for this! 👍🏼

**Comment by rubemlrm** [comment]
Sorry for the late reply :/ i've only made a optimization tip that i've read a few months ago do reduce the docker image size.

**Pull request merged**